flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/utils/RexNodeExtractor.scala

```diff
@@ -108,18 +130,15 @@ object RexNodeExtractor extends Logging {
     // converts the cnf condition to a list of AND conditions
     val conjunctions = RelOptUtil.conjunctions(cnf)
 
-    val convertedExpressions = new mutable.ArrayBuffer[Expression]
+    val convertibleRexNodes = new mutable.ArrayBuffer[RexNode]
     val unconvertedRexNodes = new mutable.ArrayBuffer[RexNode]
-    val inputNames = inputFieldNames.asScala.toArray
-    val converter = new RexNodeToExpressionConverter(
-      rexBuilder, inputNames, functionCatalog, catalogManager, timeZone)
     conjunctions.asScala.foreach(rex => {
       rex.accept(converter) match {
-        case Some(expression) => convertedExpressions += expression
```

Comment:
there is a regression here: for rexnode which can not be converted to a expression, will be treated as convertible rexnode

Reply:
no, convertible nodes are that can be converted via RexNodeToExpressionConverter